scipy/_lib/_util.py

```diff
@@ -521,6 +522,29 @@ def _fixed_default_rng(seed=1638083107694713882823079058616272161):
         np.random.default_rng = orig_fun
 
 
+def _rng_html_rewrite(func):
+    """Rewrite the HTML rendering of ``np.random.default_rng``.
+
+    This is intended to decorate
+    ``numpydoc.docscrape_sphinx.SphinxDocString._str_examples``.
+
+    Examples are only run by Sphinx when there are plot involved. Even so,
+    it does not change the result values getting printed.
+    """
+    # hexadecimal or number seed, case-insensitive
+    pattern = re.compile(r'np.random.default_rng\((0x[0-9A-F]+|\d+)\)', re.I)
```

Comment:
I didn't remember markdown syntax, so I asked ChatGPT for help. The results of
> What does the regex 'np.random.default_rng\((0x[0-9A-F]+|\d+)\)' do?

followed by
> Format that response in markdown

![image](https://user-images.githubusercontent.com/6570539/220170888-c71f7c5e-127e-4476-9618-e15256df37cc.png)

Reply:
Not bad 😅 It that could make fixes directly based on issues that would be nice.

For Regex I use this to build or analyse them http://regexr.com